src/extractors/CSVStagingExtractor.js

```diff
@@ -0,0 +1,89 @@
+const path = require('path');
+const { Extractor } = require('./Extractor');
+const { CSVModule } = require('../modules');
+const { firstEntryInBundle } = require('../helpers/fhirUtils');
+const { generateMcodeResources } = require('../templates');
+const logger = require('../helpers/logger');
+const { formatDateTime } = require('../helpers/dateUtils');
+
+function formatTNMCategoryData(stagingData) {
+  logger.debug('Reformatting TNM Category data into template format');
+  const formattedData = [];
+  const {
+    mrn, conditionId, t, m, n, type, effectiveDate,
```

Comment:
petty, almost irrelevant suggestion that would make me disproportionately happy: reordering `t` `m` and  `n` to `t`, `n`,  and `m`
```suggestion
    mrn, conditionId, t, n, m, type, effectiveDate,
```